assets/panel/debugger.properties

```diff
@@ -392,6 +392,9 @@ callStack.collapse=Collapse rows
 # message to show more of the frames.
 callStack.expand=Expand rows
 
+# LOCALIZATION NOTE (components.header): Components right sidebar pane header.
```

Comment:
Maybe something a bit more descriptive? e.g. 'Header pane in the right sidebar for React Components', assuming it's only for React

Reply:
How's this? f2c5442fcb3a64517a412e11947a7544166cd03f
'Header for the React Components pane in the right sidebar.'

For now, it's only for React.